ci/jupyter.js

```diff
@@ -104,6 +104,7 @@ class JupyterServer {
     // rx-jupyter for writing a notebook to the contents API
     const apiPath = "/api/bookstore/published/";
     const xhr = await ajax({
+      // TODO: Resolve https://github.com/nteract/bookstore/pull/75#discussion_r280785802
```

Comment:
In line with the reasoning behind bringing `url_path_join` into its own module for user clarity, I would suggest that having this utility here is simpler than introducing a new dependency (especially because as far as I can tell a dependency with this functionality does not exist).

```suggestion
```

Reply:
I'm okay with that approach. If we find down the road that we need more, we can do so later.

Reply:
I don't seem to be able to commit the suggestion to your repository, did you turn off allow edits from maintainers on this PR?